### ISSUE TITLE
configure restart of supervisor service

### ISSUE THREAD

**Issue by cehbrecht** [bird-house/ansible-wps-playbook#54]
... currently supervisor service is down after reboot.

**Comment by cehbrecht** [comment]
... or the service was not shutdown nicely:
```
[2018-12-05 07:38:41 +0100] [7009] [INFO] Starting gunicorn 19.9.0
[2018-12-05 07:38:41 +0100] [7009] [ERROR] Retrying in 1 second.
[2018-12-05 07:38:42 +0100] [7009] [ERROR] Retrying in 1 second.
[2018-12-05 07:38:43 +0100] [7009] [ERROR] Retrying in 1 second.
[2018-12-05 07:38:44 +0100] [7009] [ERROR] Retrying in 1 second.
[2018-12-05 07:38:45 +0100] [7009] [ERROR] Retrying in 1 second.
[2018-12-05 07:38:46 +0100] [7009] [ERROR] Can't connect to /var/run/pywps/c4cds.socket
```

**Comment by cehbrecht** [comment]
... the subfolder `/var/run/pywps` was removed after a restart. Maybe it is safer to skip this subfolder.

**Comment by cehbrecht** [comment]
fixed by PR #77.
